manager/src/main/java/com/redhat/service/bridge/manager/providers/ResourceNamesProviderImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.redhat.service.bridge.manager.providers;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@ApplicationScoped
+public class ResourceNamesProviderImpl implements ResourceNamesProvider {
+
+    public static final String RESOURCE_PREFIX_PROPERTY = "event-bridge.resource-prefix";
+    public static final String BRIDGE_SHORTNAME = "brdg";
+    public static final String PROCESSOR_SHORTNAME = "prcs";
+
+    private static final String VALIDATION_REGEX = "^[a-z][a-z0-9-]{0,19}$";
+
+    @ConfigProperty(name = RESOURCE_PREFIX_PROPERTY)
+    String resourcePrefix;
+
+    String validatedResourcePrefix;
+
+    @PostConstruct
+    void validate() {
+        if (resourcePrefix == null || resourcePrefix.isBlank() || !resourcePrefix.matches(VALIDATION_REGEX)) {
+            throw new IllegalArgumentException(String.format("Property \"%s\" must match the regex \"%s\"", RESOURCE_PREFIX_PROPERTY, VALIDATION_REGEX));
+        }
+        validatedResourcePrefix = resourcePrefix + (resourcePrefix.endsWith("-") ? "" : "-");
+    }
+
+    @Override
+    public String getBridgeTopicName(String bridgeId) {
+        return String.format("%s%s-%s", validatedResourcePrefix, BRIDGE_SHORTNAME, bridgeId);
+    }
+
+    @Override
+    public String getProcessorConnectorName(String processorId) {
+        return getProcessorTopicName(processorId);
```

Comment:
Surely we can generate a uniquely prefixed name for processors' connectors too?

Would it not make it easier to distinguish between the two?

Reply:
I am using the same name so the processor that the connector belongs to can immediately be identified if you're looking at the connector UI or at the connector infrastructure inside K8S.
Also there's no possibility of conflict with topics since topics and connectors belongs to two separate services (managed kafka vs managed connectors).

Reply:
I do not dispute that including the processor id in the connector's id in some form is not advantageous to help quickly correlate different artifacts. I was thinking more how to disambiguate the ids for a processor and a connector, which, if I am not mistaken is the whole point of this JIRA? My opinion is however one of many reviewing this PR. So, if I standalone so be it. Let's go with the majority opinion.

Reply:
The purpose of this PR is to make it easier to identify what an external resource (kafka topics and managed connectors at the moment) is used for.
Using the processor ID in the name of the managed connector does exactly this: give a quick link to the dev between processor and associated connector when looking at the connectors list.
It was already like this and it was ok. What this change does is also introduce the "resource prefix" in the name of the connector so that we can also identify who is creating those (developer/CI/OSD cluster).

Reply:
Since we're talking about this, we're missing the other way around (having the connectorId in the processor) it's covered by https://issues.redhat.com/browse/MGDOBR-474